Sources_v3/APIClient/Endpoints/ChannelEndpoints.swift

```diff
@@ -117,4 +117,14 @@ extension Endpoint {
             body: nil
         )
     }
+    
+    static func markAllRead() -> Endpoint<EmptyResponse> {
+        .init(
+            path: "channels/read",
+            method: .post,
+            queryItems: nil,
+            requiresConnectionId: false,
+            body: nil
+        )
+    }
```

Comment:
missing tests

Reply:
Fixed in 00fda7371027455896dc395d5e012eca690580ef